### ISSUE TITLE
Update Setup Documentation

### ISSUE THREAD

**Issue by jbkalmbach** [DarkEnergyScienceCollaboration/Twinkles#350]
I guess I missed this in the recent pull requests, but it now seems generatePhosimInput.py has been deprecated in favor of twinkles_sky.py. Can we update https://github.com/DarkEnergyScienceCollaboration/Twinkles/blob/master/doc/Setup.md to reflect new usage @rbiswas4?

**Comment by jbkalmbach** [comment]
@rbiswas4 If it's just that easy as replacing one script in favor of the other, let me know and I'll test it out and write up my steps if you want.

**Comment by rbiswas4** [comment]
Yes, this is correct. I did write an example along with the cookbook : in this PR: https://github.com/DarkEnergyScienceCollaboration/Twinkles/pull/328

@jchiang87 made a good point about adding database setup variables, and then we realized that this is actually @danielsf 's cookbook. (So I think he plans to do a check before this change and maybe the fileDB change is added?)

**Comment by jbkalmbach** [comment]
Ok, I'll assign this to myself then. Also, since we add in all the variable elements in the same script I propose we just have "Sprinkling Setup" instead of headers for both SL and SN.

**Comment by drphilmarshall** [comment]
Sounds good. Thanks Bryce!

**Comment by jbkalmbach** [comment]
After reading through the sims_cookbook @danielsf and @rbiswas4 wrote up, I think that just by sending people to that we have the desired instructions already.

**Comment by danielsf** [comment]
I think there is unique information in `Setup.md` (for instance, how to create `$HOME/.lsst/db-auth.paf` for connection to fatboy).  If we really want to get rid of `Setup.md`, we should make sure to merge that information into one of the cookbooks.

**Comment by danielsf** [comment]
Okay.  I misread your original comment from last night.  That sounds fine to me
